src/backend/parser/parse_agg.c

```diff
@@ -1505,16 +1536,138 @@ get_groupclause_exprs(Node *grpcl, List *targetList)
 		List *exprs = (List *)grpcl;
 		ListCell *lc;
 
-		foreach (lc, exprs)
+		foreach(lc, exprs)
 		{
-			result = list_concat(result, get_groupclause_exprs((Node *)lfirst(lc),
+			result = list_concat(result, get_groupclause_tles((Node *)lfirst(lc),
 															   targetList));
 		}
 	}
 
 	return result;
 }
 
+static List*
```

Comment:
Please add comments describing what this function does

Reply:
Added